setup.py

```diff
@@ -27,7 +27,7 @@
 
 setup(
     name='tx_salaries',
-    version='1.8.0',
+    version='1.9.0',
```

Comment:
I bump this when I push out an update, so we shouldn't leave this in here when we merge in changes.